scripts/main.js

```diff
@@ -31,10 +32,14 @@ projectPopUp.exitBtn.addEventListener(('click'), () => {
   projectPopUp.closePopUp();
 });
 
-window.addEventListener('load', initProjects());
+window.addEventListener('load', () => {
+  initProjects();
+  loadInputData();
+});
```

Comment:
Hello! @vichuge
I do load the data on load using the 'load' event. It works the same as specified in the [official docs](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event)

Reply:
Yep, you're right, that was my bad. Just ask again for a CR and then the next code reviewer should accept your project.